gazebo_plugins/src/gazebo_ros_imu.cpp

```diff
@@ -99,6 +99,15 @@ void GazeboRosIMU::LoadThread()
   else
     this->link_name_ = this->sdf->Get<std::string>("bodyName");
 
+  if (!this->sdf->HasElement("frameId"))
+  {
+    ROS_INFO("imu plugin missing <frameId>, defaults to 'imu'");
+    this->frame_id_ = "imu";
+    return;
+  }
+  else
+    this->frame_id_ = this->sdf->Get<std::string>("frameId");
```

Comment:
Personal preference, but could we wrap the else block in curly braces? Obviously it's syntactically fine either way, I just hate forgetting to add the braces when I want to add more lines to a block.